### PR TITLE
Sqlite fixes

### PR DESCRIPTION
**Pull request by dmeltzer**
This is a better version of my previous pull request related to sqlite fixes.  It doesn't contain the importer changes (oops), and puts username back into the list.  I'm still unsure that managerList() is doing what it should, but I don't know if I broke that..

### CODE REVIEW

**Comment by dmeltzer** [comment]
Updated to remove remaining concat() usage.  This has the benefit of removing a lot of duplicated code in AssetMantainenceController and LicenseController, and moving a method to the Asset Model to return a formatted string of "Asset Id - Asset Name (Checked Out To) Model Name", which reduces some large queries.

File: app/Models/Asset.php [hunk 1]
```diff
@@ -172,6 +172,15 @@ public function checkOutNotifySlack($settings, $admin, $note = null)
     }
 
 
+    public function getDetailedNameAttribute() {
+      $user = $this->assigneduser;
+      if($user) {
```
Comment (snipe) [inline app/Models/Asset.php]: Could just do:

`if($this->assigneduser) {`

File: app/Models/User.php [hunk 2]
```diff
@@ -118,6 +118,10 @@ public function fullName()
         return "{$this->first_name} {$this->last_name}";
     }
 
+    public function getFullNameAttribute()
```
Comment (snipe) [inline app/Models/User.php]: How is this different than `fullName`?

File: app/Models/Asset.php (hunk 1 above)
Comment (dmeltzer) [inline app/Models/Asset.php]: Fixed.  I think I was debugging something at some point and then deleted it.

File: app/Models/User.php (hunk 2 above)
Comment (dmeltzer) [inline app/Models/User.php]: Functionally it's the same, the difference is that laravel magic makes a  'full_name' property available as a value in an Eloquent Collection this way, so things like lists() function.  We could either make getDetailedNameAttribute() return fullName(), or modify everywhere that uses ->fullName() currently to instead call ->full_name, and remove fullName().  I'm not sure which is a better approach.

**Comment by vjandrea** [comment]
+1 for @dmeltzer's approach to use full_name as an attribute everywhere, so we can kiss concat() goodbye [sqlite user here]

**Comment by snipe** [comment]
Is this ready for merging?

**Comment by dmeltzer** [comment]
Yes.  I have not encountered any other incompatabilities.  I haven't done a huge amount of testing with mysql, but the testing I have done has not seen any issues.

**Comment by dmeltzer** [comment]
Rebased onto current v3.

**Pull request merged**